### DISCUSSION TITLE
Schema and VSchema

### DISCUSSION THREAD

**Discussion by puchesjr** [planetscale/discussion#41] [Q&A] [answered]
Looking a the Vitess documentation, do we need to create VSchema or will the DDL suffice?

**Comment by Anders-PlanetScale** [comment, marked as answer]
Hello @puchesjr,

The current beta version of PlanetScale DB does not support database sharding. This is on our road-map but won't be part of the beta.
So you won't need to worry about creating any VSchema at this point in time.

**Reply by mvandeweerd** [reply]
Hi. Is there an expected date for sharding to go live?

**Reply by jgreet** [reply]
There isn't a date we can share for self-serve database sharding. We can shard customer databases on PlanetScale right now when necessary but it requires some coordination with our engineers.
